src/main/java/com/bankxpvalue/BankXpValueConfig.java

```diff
@@ -26,10 +26,10 @@ default boolean showTutorial(){
     @ConfigItem(
             position = 2,
             keyName = "keepFixed",
-            name = "Lock overlay in center",
-            description = "Keeps the overlay fixed in the center"
+            name = "Reset overlay to center",
+            description = "Resets the overlay to the center of the bank window when shown"
```

Comment:
Do you know how if you hold Alt (or Option on mac) + right click an overlay, it resets to the default position? I'm almost thinking we should remove this config option and just implement the alt + right click event handler so that it maintains that behavior/consistency. What do you think?

Reply:
Oh that's a neat idea. Ya that behavior doesn't exist as far as I know. I'm not sure if you can get click events on your overlay though can you?

Reply:
I mean you should be able to considering the right click reset already exists, but I'm not sure. The config options works fine if not.

Reply:
Sorry, what do you mean the right click reset already exists? I'm not following 😅